impeller/core/formats.h

```diff
@@ -293,6 +293,13 @@ constexpr const char* TextureUsageToString(TextureUsage usage) {
 
 std::string TextureUsageMaskToString(TextureUsageMask mask);
 
+enum class UsageHint {
+  /// @brief Texture or buffer is being used during the raster workload.
+  kRasterWorkload,
```

Comment:
How about `kTransient` and `kPermanent`? You expect to allocate the former in or near a frame workload and the latter has a longer lifetime.

Reply:
My sense is that the current names betray this being used for a very Flutter as it works today use case.